csm/cli/schema/health.json

```diff
@@ -0,0 +1,87 @@
+{
+    "name": "health",
+    "description": "Health",
+    "sub_commands": [
+      {
+        "name": "show",
+        "description": "Displays health of the cluster and resources in it.",
+        "need_confirmation": false,
+        "permissions_tag": "list",
+        "args": [
+          {
+            "flag": "resource",
+            "type": "str",
+            "help": "The type of element in the hierarchy. Ex. node"
+          },
+          {
+            "flag": "-i",
+            "dest": "resource_id",
+            "type": "str",
+            "help": "Id of the resource.",
+            "default": "",
+            "params": true
+          },
+          {
+            "flag": "-o",
+            "dest": "offset",
+            "type": "int",
+            "help": "The (page) number from which the records should be fetched",
+            "default": "1",
+            "params": true
+          },
+          {
+            "flag": "-l",
+            "dest": "limit",
+            "type": "int",
+            "help": "The number of records to be fetched.",
+            "default": "1",
+            "params": true
+          },
+          {
+            "flag": "-r",
+            "dest": "response_format",
+            "type": "str",
+            "default": "table",
+            "suppress_help": true,
+            "help": "",
+            "params": true
```

Comment:
How is different from `-f` flag? `help` is suppressed ..user wont able to find difference between `-r` and `-f` flags

Reply:
@udayan-y21, 
REST api is strictly expecting a query param : response_format=table. This param will return response data in single array which is required for tabular format view. And user is not allowed to change the default value of this param that is why suppress_help is set to true.